libs/telemetry/src/main/java/org/opensearch/telemetry/tracing/Level.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.telemetry.tracing;
+
+import java.util.Arrays;
+
+/**
+ * The Level class defines a set of standard tracing levels that can be used to control tracing output.
+ * The tracing Level objects are ordered and are specified by ordered integers.
+ * Enabling tracing at a given level also enables tracing at all higher levels.
+ *
+ * Levels in descending order are
+ * <ul>
+ *  <li>ROOT(highest value)</li>
+ *  <li>TERSE</li>
+ *  <li>INFO</li>
+ *  <li>DEBUG</li>
+ *  <li>TRACE(lowest value)</li>
+ * </ul>
+ *
+ * In addition, there is a level DISABLED that can be used to turn off tracing. By default, tracing is disabled.
+ *
+ */
+public enum Level {
+    /**
+     * DISABLED is a special level that can be used to turn off tracing. This level is initialized to Integer.MAX_VALUE.
+     */
+    DISABLED(Byte.MAX_VALUE),
```

Comment:
If tracing is disabled, it should be set on trace level, not span level, please remove it

Reply:
Actually I too was double minded where to put it. Made this a tracer level setting now